setup.cfg

```diff
@@ -2,7 +2,7 @@
 looponfailroots = tests
 python_classes = Test Describe
 python_files = test_*.py
-python_functions = test_ it_ they_ but_ and_it_ or_it
+python_functions = test_ it_ they_ but_ and_it_ or_it and_
```

Comment:
I believe that the `and_` is not necessary. We should use the verbose `and_it` and so the new tests should be `and_it_does_not_xxxxxx`